core/src/actions/exports.ts

```diff
@@ -108,3 +108,32 @@ export class ExportView extends AuthenticatedAction {
     return { export: await _export.apiData() };
   }
 }
+
+export class ExportsRetryFailed extends AuthenticatedAction {
+  name = "exports:retryFailed";
+  description = "retry failed exports within a timeframe";
+  permission: ActionPermission = { topic: "destination", mode: "write" };
+  inputs = {
+    destinationId: { required: false },
+    startTimestamp: { required: true, formatter: APIData.ensureNumber },
+    endTimestamp: { required: true, formatter: APIData.ensureNumber },
+    preview: { required: false, formatter: APIData.ensureBoolean },
```

Comment:
[nit] maybe a default value of `false` would be more clear

Reply:
done in https://github.com/grouparoo/grouparoo/pull/2825/commits/6d61192a3b80a08f8b0510cfc1e3dfc2860a00b4